integration/local-gadget/Makefile

```diff
@@ -66,17 +71,15 @@ test: test-$(CONTAINER_RUNTIME)
 # INTEGRATION_TESTS_PARAMS can be used to pass additional parameters locally e.g
 # INTEGRATION_TESTS_PARAMS="-test.run TestListContainers" CONTAINER_RUNTIME=containerd make -C integration/local-gadget test
 .PHONY: phony_explicit
-test-%: build
+test-%: build build-local-gadget-tests
 	export MINIKUBE_PROFILE=minikube-$* && \
 	echo "Checking minikube with profile $${MINIKUBE_PROFILE} is running ..." && \
 	$(MINIKUBE) status -p $${MINIKUBE_PROFILE} -f {{.APIServer}} >/dev/null || (echo "Error: $${MINIKUBE_PROFILE} not running, exiting ..." && exit 1) && \
 	echo "Preparing minikube with profile $${MINIKUBE_PROFILE} for testing ..." && \
 	$(MINIKUBE) cp ../../local-gadget-linux-amd64 $${MINIKUBE_PROFILE}:/bin/local-gadget >/dev/null && \
 	$(MINIKUBE) ssh sudo chmod +x /bin/local-gadget && \
-	go test -c -o local-gadget-integration.test ./... && \
 	$(MINIKUBE) cp local-gadget-integration.test $${MINIKUBE_PROFILE}:/bin/local-gadget-integration.test >/dev/null && \
 	$(MINIKUBE) ssh sudo chmod +x /bin/local-gadget-integration.test && \
-	rm local-gadget-integration.test && \
```

Comment:
shouldn't we keep this? or am i missing something?

Reply:
I don't why we would or want to remove the text executable.
If it is there we don't need to build or copy it

Reply:
Mmmmm unless I am missing something, there is no other way to be sure that the existing `local-gadget-integration.test` is using the latest code than building `local-gadget-tests` again, and it will overwrite `local-gadget-integration.test`. Therefore, I don't see the advantage of keeping it.

Reply:
Hmmm, makes sense.
There is no Makefile or other relationship in the build chain from `local-gadget-integration.test` to the source files